Formula/liblzf.rb

```diff
@@ -22,6 +22,8 @@ class Liblzf < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "04201793eebd3ab8a64694c1422e4d5801be13fd26d1ed257b6c31d4a231124c"
   end
 
+  conflicts_with "harbour", because: "both install `liblzf.a` static libraries"
```

Comment:
May want to add conflicts in separate `CI-syntax-only` PR to avoid rebuilding bottles.

Reply:
Sounds like a good idea

Reply:
Yes, actually not only do I need to do that in a `CI-syntax-only` script, but I realized I need to fix the `libxdiff` flat namespace issue separately first and rebase this.